### DISCUSSION TITLE
elastic-agent unhealthy packet-default input not supported

### DISCUSSION THREAD

**Discussion by andreascccathens** [Security-Onion-Solutions/securityonion#11757] [2.4]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

100gb

### Storage for /nsm

400gb

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

the network packet capture v1.19.1 integration on the elastic agent for windows server 2019, 2022 fails with error "input not supported"

Spawned new component packet-default: input not supported","log":{"source":"elastic-agent"},"component":{"id":"packet-default","state":"FAILED"},"ecs.version":"1.6.0"}
Spawned new component packet-default: input not supported","log":{"source":"elastic-agent"},"component":{"id":"packet-default","state":"FAILED"},"ecs.version":"1.6.0"}
{"log.level":"error","@timestamp":"2023-11-10T08:38:58.033+0200","log.origin":{"file.name":"coordinator/coordinator.go","file.line":978},"message":"Spawned new unit packet-default-packet-network-f40cbb90-7d68-11ee-a98a-6d7be746bbae: input not supported","log":{"source":"elastic-agent"},"component":{"id":"packet-default","state":"FAILED"},"unit":{"id":"packet-default-packet-network-f40cbb90-7d68-11ee-a98a-6d7be746bbae","type":"input","state":"FAILED"},"ecs.version":"1.6.0"}

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by cm-ops** [comment]
Here is a list of the current supported integrations - https://docs.securityonion.net/en/2.4/elastic-fleet.html#integrations

**Reply by andreascccathens** [reply]
didnt realize thanks Chris

**Reply by andreascccathens** [reply]
BTW updated to 2.4.30 and agent 8.10.4 and it works now
